tests/test_models/test_lithium_ion.py

```diff
@@ -0,0 +1,75 @@
+#
```

Comment:
I would suggest splitting this function up into individual functions once the tests get more involved